code/game/area/areas.dm

```diff
@@ -188,6 +177,22 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 		map_generator = new map_generator()
 		map_generator.generate_terrain(get_area_turfs(src))
```

Comment:
Shouldn't lines 176-178 be removed? Since terrain generation no longer happens here, and it's like that in the upstream PRs you ported (this is where the runtime happens that breaks CI)

Reply:
With those removed it runs great